starfish/experiment/experiment.py

```diff
@@ -89,11 +107,69 @@ def name(self) -> str:
     def image_types(self) -> Set[str]:
         return set(self._images.keys())
 
-    def get_image(self, item: str, crop_parameters: Optional[CropParameters]=None) -> ImageStack:
-        return ImageStack.from_tileset(self._images[item], crop_parameters)
+    def show_aligned_image_groups(self):
+        """
+        Describe the aligned subgroups for each Tileset in this FOV
+
+        ex.
+            {'primary': 'Group 0:  <starfish.ImageStack (8, 2, 1, 205, 405))>',
+             'nuclei': 'Group 0:  <starfish.ImageStack (1, 1, 1, 205, 405))>'}
+
+        Means there are two tilesets in this FOV, (primary and nuclei) each tileset only
```

Comment:
Here I would state the reason why there is only one TileSet group to make this a bit clearer: 

`Means there are two tilesets in this FOV (primary and nuclei), and because all images have the same (x, y) coordinates, each tileset has a single aligned subgroup.`

Then, below:

`Means there is one tileset in this FOV (primary), but the 4 rounds of Group 0 have different (x, y) coordinates from the rounds of Group 1, and the tiles are therefore partitioned into different coordinate groups.`

Also, with the `repr` the way it is, it's not possible to disambiguate in this example whether the cause of the misalignment is over round (n=8) or channel (n=4). Could it make sense to print out the channels and rounds associated with each Group? e.g: 

```
            {'primary': 'Group 0:  <starfish.ImageStack round=(0, 1, 2, 3), ch=(0, 1), (x, y, z)=(1, 205, 405)>
                         Group 1:  <starfish.ImageStack round=(4, 5, 6, 7), ch=(0, 1), (x, y, z)=(1, 205, 405)>'}
```